scripts/config.py

```diff
@@ -25,6 +25,7 @@ class Config(metaclass=Singleton):
 
     def __init__(self):
         self.debug = False
+        self.script_path = "input/sample_script.yml"
```

Comment:
What's the advantage of not always using a script named something like `input/script.yml`?